dxf doc change on 8.1 (#17409)
pingcap/docs: tidb-global-sort.md

To enable Global Sort, follow these steps:

> 1. Enable the DXF by setting the value of [`tidb_enable_dist_task`](/system-variables.md#tidb_enable_dist_task-new-in-v710) to `ON` (this variable is enabled by default starting from v8.1.0, so you can skip this step if your cluster is newly created):

Reviewer: ```suggestion
1. Enable the DXF by setting the value of [`tidb_enable_dist_task`](/system-variables.md#tidb_enable_dist_task-new-in-v710) to `ON`. Starting from v8.1.0, this variable is enabled by default. For newly created clusters of v8.1.0 or later versions, you can skip this step:
```
